tweet-api/api/app.py

```diff
@@ -0,0 +1,73 @@
+import logging
+import time
+from logging.config import dictConfig
+from flask import Flask, request
+from flask_cors import CORS
+
+from api import inference
+from api.utils import _NumpyEncoder
+
+
+dictConfig(
+    {
+        "version": 1,
+        "formatters": {
+            "default": {
+                "format": "[%(asctime)s] %(levelname)s in %(module)s: %(message)s",
+            }
+        },
+        "handlers": {
+            "wsgi": {
+                "class": "logging.StreamHandler",
+                "stream": "ext://flask.logging.wsgi_errors_stream",
+                "formatter": "default",
+            }
+        },
+        "root": {"level": "INFO", "handlers": ["wsgi"]},
+    }
+)
+
+logging.getLogger("flask_cors").level = logging.DEBUG
+app = Flask(__name__)
+CORS(app)
+
+app.json_encoder = _NumpyEncoder
+
+
+@app.route("/")
+def hello_world():
+    return "<p>Hello, World!</p>"
```

Comment:
Suggestion: use the same kind of banner as I do for Dockercoins (https://github.com/jpetazzo/container.training/blob/main/dockercoins/rng/rng.py#L18) because it shows the container/pod identity